src/sentry/middleware/integrations/classifications.py

```diff
@@ -8,24 +8,9 @@
 from django.http import HttpRequest
 from django.http.response import HttpResponseBase
 
-from .parsers import (
-    BitbucketRequestParser,
-    BitbucketServerRequestParser,
-    GithubEnterpriseRequestParser,
-    GithubRequestParser,
-    GitlabRequestParser,
-    JiraRequestParser,
-    JiraServerRequestParser,
-    MsTeamsRequestParser,
-    PluginRequestParser,
-    SlackRequestParser,
-    VstsRequestParser,
-)
-
 if TYPE_CHECKING:
     from sentry.middleware.integrations.integration_control import ResponseHandler
-
-    from .parsers.base import BaseRequestParser
```

Comment:
Since the classifications will now be read at app initalization (to add the getsentry ones), we need to move around some imports.